modules/dnn/src/init.cpp

```diff
@@ -209,6 +209,7 @@ void initializeLayerFactory()
     CV_DNN_REGISTER_LAYER_CLASS(FlowWarp,       FlowWarpLayer);
 
     CV_DNN_REGISTER_LAYER_CLASS(LSTM,           LSTMLayer);
+    CV_DNN_REGISTER_LAYER_CLASS(LSTM2,          LSTM2Layer);
```

Comment:
That is the first layer named in such a way. I think that layer name should remain the same while `Impl` class should be different for different versions of the engine

Reply:
Not first. here are more examples [[1](https://github.com/opencv/opencv/blob/979428d5908b5396237b18cbd50d0b18265e2463/modules/dnn/src/init.cpp#L92)], [[2](https://github.com/opencv/opencv/blob/979428d5908b5396237b18cbd50d0b18265e2463/modules/dnn/src/init.cpp#L88)], [[3](https://github.com/opencv/opencv/blob/979428d5908b5396237b18cbd50d0b18265e2463/modules/dnn/src/init.cpp#L107)]

Reply:
Oh, I see.